src/sagemaker_mxnet_container/serving.py

```diff
@@ -0,0 +1,273 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json
+import logging
+import os
+
+import mxnet as mx
+from sagemaker_containers.beta.framework import encoders, env, modules, transformer, worker
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_MODEL_NAME = 'model'
+DEFAULT_MODEL_FILENAMES = {
+    'symbol': 'model-symbol.json',
+    'params': 'model-0000.params',
+    'shapes': 'model-shapes.json',
+}
+
+
+def default_model_fn(model_dir, preferred_batch_size=None):
+    """Function responsible for loading the model. For more information, see
+    https://github.com/aws/sagemaker-python-sdk#model-loading.
+
+    Args:
+        model_dir (str): The directory where model files are stored.
+
+    Returns:
+        mxnet.mod.Module: the loaded model.
+    """
+    for f in DEFAULT_MODEL_FILENAMES.values():
+        path = os.path.join(model_dir, f)
+        if not os.path.exists(path):
+            raise ValueError('missing %s file' % f)
```

Comment:
I would like a bit more context with the error message - something like:

failed to load model with default_model_fn: missing %s file....

Reply:
done